src/SerialLoops/Editors/DialogueConfigEditor.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Eto.Forms;
+using HaruhiChokuretsuLib.Archive.Event;
+using HaruhiChokuretsuLib.Util;
+using SerialLoops.Lib.Items;
+using SerialLoops.Utility;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SerialLoops.Editors
+{
+    public class DialogueConfigEditor : Editor
+    {
+        private DialogueConfigItem _dialogueConfig;
+
+        public DialogueConfigEditor(DialogueConfigItem dialogueConfig, ILogger log) : base(dialogueConfig, log)
+        {
+        }
+
+        public override Container GetEditorPanel()
+        {
+            _dialogueConfig = (DialogueConfigItem)Description;
+
+            DropDown characterDropDown = new();
+            characterDropDown.Items.AddRange(Enum.GetNames<Speaker>().Select(c => new ListItem { Key = c.ToString(), Text = c.ToString() }));
+            characterDropDown.SelectedKey = _dialogueConfig.DialogueConfig.Character.ToString();
+
+            return new StackLayout
+            {
+                Spacing = 5,
+                Items =
+                {
+                    ControlGenerator.GetControlWithLabel("Character", characterDropDown),
+                    ControlGenerator.GetControlWithLabel("Voice Font", new TextBox { Text = _dialogueConfig.DialogueConfig.VoiceFont.ToString() }),
+                    ControlGenerator.GetControlWithLabel("Text Timer", new TextBox { Text = _dialogueConfig.DialogueConfig.TextTimer.ToString() }),
```

Comment:
Perhaps NumericSteppers would be better here, too?
```suggestion
                    ControlGenerator.GetControlWithLabel("Voice Font", new NumericStepper { Value = _dialogueConfig.DialogueConfig.VoiceFont }),
                    ControlGenerator.GetControlWithLabel("Text Timer", new NumericStepper { Value = _dialogueConfig.DialogueConfig.TextTimer }),
```

Reply:
Oooh yes! That's perfect.